lib/brainstem/presenter_collection.rb

```diff
@@ -71,8 +71,16 @@ def structure_response(presented_class, primary_models, count, options)
       selected_associations = filter_includes(options)
 
       optional_fields = filter_optional_fields(options)
+      page_size = calculate_per_page(options)
 
-      struct = { 'count' => count, brainstem_key => {}, 'results' => [] }
+      struct = {
+        'count' => count,
+        'page_number' => count > 0 ? options[:params].fetch(:page, 1).to_i : 0,
```

Comment:
If there are no results, should the page_number be 0, 1, or does it matter?

Reply:
I think `0` makes the most sense. It's the same dealio with `count`. There are 0 pages, there are 0 results.